package.json

```diff
@@ -33,6 +34,7 @@
 		"@types/moment": "^2.13.0",
 		"class-variance-authority": "^0.7.0",
 		"clsx": "^2.1.1",
+		"codecache": "file:",
```

Comment:
why there is a change in package.json and package-lock.json, kindly revert it back